docs/community/bindings.md

```diff
@@ -9,6 +9,7 @@ As a C library, bindings can be made to call H3 functions from different program
 ## C&#35;
 
 - [entrepreneur-interet-general/h3.standard](https://github.com/entrepreneur-interet-general/H3.Standard)
+- [richardvasquez/h3net](https://github.com/RichardVasquez/h3net)
```

Comment:
Might be useful to note this is a translation rather than a binding for those interested. Not totally necessary, we only have two links. :)

Reply:
Noted and updated